master/buildbot/config.py

```diff
@@ -21,6 +21,7 @@
 
 from future.utils import iteritems
 from future.utils import itervalues
+from future.utils import string_types
```

Comment:
future.utils.string_types does the job quite nicely here, and I am confident that its usage
here is correct.  The past module is to be avoided and used sparingly.  The past module is meant for people who have huge codebases and don't want to make lots of changes.

The strategy that I am using is to bring the buildbot codebase forward while still preserving behavior and not breaking things on Python 2.6/2.7.